tasks/setup.js

```diff
@@ -0,0 +1,51 @@
+module.exports = function() {
+  // Global variables
+  var plugins = this.opts.plugins,
+      config  = this.opts.configs,
+      fs      = require('fs-extra'),
```

Comment:
We should add this in `gulpfile.js` and map `fs-extra` to `fs` and then use as a `plugins.fs` across the project.

BTW. It's werid, why they not merge this to node `js` package...